templates/publishing/src/blog-app.html

```diff
@@ -40,6 +40,7 @@
 
       :host {
         --app-primary-color: #404040;
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```

Comment:
IIRC `transparent` is required for some Android phones/browsers. If it works on the most recent Chrome though, I think it's okay to remove it.